lib/model-access-routes.js

```diff
@@ -43,6 +43,7 @@ class ModelAccessRoutes {
 		if (!options.routePrefix) {
 			options.routePrefix = decamelize(this.model.getName(), '-');
 		}
+		options.subschemaOptions = { ignoreDefaults: true };
```

Comment:
Instead of subschemaOptions, call this something like documentSchemaOptions or modelSchemaOptions.  These options don't apply to all subschemas, only the model ones.